### PR TITLE
Verify iPhone sensor data sanity

### PR DESCRIPTION
**Pull request by mpetruska**
https://github.com/eigengo/lift/issues/75

Updated ios code to include the inferred gravity value, so that the whole measured vector is tranferred.

### CODE REVIEW

**Comment by mpetruska** [comment]
Updated documentation to reflect the measurement unit of rotation more precisely.

**Comment by mpetruska** [comment]
Ready for review.

File: ios/Lift/Device/ByteVector.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            var x: Int16 = 0
+            var y: Int16 = 0
+            var z: Int16 = 0
+            let pointerAtOffset = UnsafePointer<()>(pointer.baseAddress).advancedBy(self.currentPosition)
+            decode_lift_accelerometer_data(pointerAtOffset, &x, &y, &z)
+            return [x, y, z]
+        }
+        currentPosition += 5
+        return result
+    }
```
Comment (carlpulley) [inline ios/Lift/Device/ByteVector.swift]: Why do we increment by 5 in this code? We read 3 values, and 1 value is an Int16, so when byte reading, surely we need to increment by 2*3 = 6?

Comment (mpetruska) [inline ios/Lift/Device/ByteVector.swift]: We read a packed structure that is 5 bytes long, see: SensorDataProtocol.h or gfs.h.

**Comment by carlpulley** [comment]
LGTM, :+1:

File: ios/Lift/Device/SensorData.swift [hunk 2]
```diff
@@ -124,6 +124,13 @@ class SensorDataGroup {
         }
     }
     
+    private func _debug_printMinMaxValues(type: UInt8, samples: NSData) {
```
Comment (zapletal-martin) [inline ios/Lift/Device/SensorData.swift]: Does not seem to be used (feel free to keep if required though).

**Comment by zapletal-martin** [comment]
:+1:

File: ios/Lift/Device/SensorData.swift (hunk 2 above)
Comment (mpetruska) [inline ios/Lift/Device/SensorData.swift]: Yes, it is unused now, will help in debugging sensor data values in the future.

**Pull request merged**